frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -323,302 +323,328 @@ function removeForestClientFromList(index: number) {
 </script>
 
 <template>
+    <PageTitle
+        title="Add user permission"
+        subtitle="Add a new permission to a user. All fields are mandatory unless noted"
+    />
     <VeeForm
         ref="form"
         v-slot="{ errors, meta }"
         :validation-schema="formValidationSchema"
         as="div"
     >
-        <PageTitle
-            title="Grant access to a user"
-            :subtitle="'You are editing in ' + selectedApplicationDisplayText"
-        />
         <div class="page-body">
-            <div class="row">
-                <form id="grantAccessForm" class="form-container">
-                    <div class="row">
-                        <label> Select user's domain </label>
-                        <div class="px-0">
-                            <RadioButton
-                                v-model="formData.domain"
-                                :checked="isIdirDomainSelected()"
-                                inputId="idirSelect"
-                                name="domainRadioOptions"
-                                :value="domainOptions.IDIR"
-                                @change="userDomainChange()"
+            <form id="grantAccessForm" class="form-container">
+                <div class="user-radio-group">
+                    <StepInfo
```

Comment:
Let's refactor this StepInfo component. 
- Rename it to StepContainer and move it under the grant access folder
- Update the StepContainer component content to be:
```
<div class="stepGroup">
         <h1 class="title">{{ props.title }}</h1>
         <h2 class="subtitle">{{ props.subtitle }}</h2>
         <slot /> // this will be the grant access step content we pass to it later
         <Divider /> // in the case the last step doesn't need the divider, can make this optional by using a parameter and set default value to true to show the divider by default
</div>
```
- So in this GrantAccess component, the structure should look like this:
```
<form id="grantAccessForm" class="form-container">
    <StepContainer title="" subtitle="">
           .....// user information fields code
    </StepContainer>
    <StepContainer title="" subtitle="">
           .....// role selection fields code
    </StepContainer>
    <StepContainer title="" subtitle="">
           .....// organization info fields code
    </StepContainer>
    .... // the reset cancel, submit buttons
</form>
```

Let's clean this up a little bit, thanks!!

Reply:
Thanks, cleaned the code as suggested!